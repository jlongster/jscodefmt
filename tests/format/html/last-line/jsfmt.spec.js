run_spec(__dirname, ["html"], { angleBracketSameLine: true });
run_spec(__dirname, ["html"], { angleBracketSameLine: false });
