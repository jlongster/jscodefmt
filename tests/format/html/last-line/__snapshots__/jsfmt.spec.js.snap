// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`brackets-on-next-line.html - {"angleBracketSameLine":false} format 1`] = `
====================================options=====================================
angleBracketSameLine: false
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit;"
>
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit;"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur recusandae error
    molestiae vero obcaecati delectus velit esse exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>
=====================================output=====================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit"
>
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit.
    <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur
    recusandae error molestiae vero obcaecati delectus velit esse
    exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>

================================================================================
`;

exports[`brackets-on-next-line.html - {"angleBracketSameLine":true} format 1`] = `
====================================options=====================================
angleBracketSameLine: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit;"
>
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit;"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur recusandae error
    molestiae vero obcaecati delectus velit esse exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>
=====================================output=====================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit">
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit">
    Lorem ipsum dolor sit amet consectetur adipisicing elit.
    <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur
    recusandae error molestiae vero obcaecati delectus velit esse
    exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>

================================================================================
`;

exports[`brackets-on-same-line.html - {"angleBracketSameLine":false} format 1`] = `
====================================options=====================================
angleBracketSameLine: false
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit;">
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit;">
    Lorem ipsum dolor sit amet consectetur adipisicing elit. <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur recusandae error
    molestiae vero obcaecati delectus velit esse exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>
=====================================output=====================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit"
>
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit.
    <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur
    recusandae error molestiae vero obcaecati delectus velit esse
    exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>

================================================================================
`;

exports[`brackets-on-same-line.html - {"angleBracketSameLine":true} format 1`] = `
====================================options=====================================
angleBracketSameLine: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit;">
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit;">
    Lorem ipsum dolor sit amet consectetur adipisicing elit. <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur recusandae error
    molestiae vero obcaecati delectus velit esse exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>
=====================================output=====================================
<div
  id="lorem-ipsum-dolor-sit-amet"
  class="consectetur-adipisicing-elit"
  title="amet deleniti illo aliquam dolore"
  style="background: inherit; color: inherit">
  <p
    id="laboriosam-eaque-tenetur-recusandae"
    class="error-molestiae-vero-obcaecati"
    title="delectus velit esse exercitationem, nihil quam quibusdam"
    style="background: inherit; color: inherit">
    Lorem ipsum dolor sit amet consectetur adipisicing elit.
    <strong>Amet deleniti illo aliquam dolore</strong> laboriosam eaque tenetur
    recusandae error molestiae vero obcaecati delectus velit esse
    exercitationem, nihil quam quibusdam iure tempore!
  </p>
</div>

================================================================================
`;
